lib/active_admin/exception_app.rb

```diff
@@ -0,0 +1,79 @@
+require 'active_admin/view_helpers'
+
+module ActiveAdmin
+  class ExceptionController < ActionController::Base
+    include Rails.application.routes.url_helpers
+    helper ::ActiveAdmin::ViewHelpers
+
+    def self.call(env)
+      response = action(:index)
+      response.call(env)
+    end
+
+    def index
+    end
+
+
+    def active_admin_application
+      ActiveAdmin.application
+    end
+
+    helper_method :active_admin_application
+
+    def active_admin_namespace
+      namespace = active_admin_application.default_namespace
+      active_admin_application.namespace(namespace)
+    end
+
+    helper_method :active_admin_namespace
+
+
+    def active_admin_config
+      ::ActiveAdmin::Page.new(active_admin_namespace, "Error", {})
+    end
+
+    helper_method :active_admin_config
+
+    def current_menu
+      active_admin_config.navigation_menu
+    end
+
+    helper_method :current_menu
+
+    def env
+      request.env
+    end
+
+    helper_method :env
+
+    def authorized?(action, subject = nil)
+      true
+    end
+
+    helper_method :authorized?
+
+    def current_active_admin_user
+      send(active_admin_namespace.current_user_method) if active_admin_namespace.current_user_method
+    end
+
+    helper_method :current_active_admin_user
+
+    def current_active_admin_user?
+      !!current_active_admin_user
+    end
+
+    helper_method :current_active_admin_user?
+
+    def destroy_admin_user_session
+      active_admin_namespace.logout_link_path
+    end
+
+    helper_method :destroy_admin_user_session
+
+    def status
+      request.env['STATUS']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.